packages/pycharm-community-bin/pycharm-community-bin.pacscript

```diff
@@ -0,0 +1,31 @@
+name="pycharm-community-bin"
+version="2021.3.3"
+pkgname="pycharm-community"
+gives="pycharm-community"
+maintainer="Obsidian <emil-aarestrup@outlook.dk>"
+description="Python IDE for professional developers aimed at productivity with a keyboard-centric focus."
+url="https://download.jetbrains.com/python/pycharm-community-${version}.tar.gz"
+hash="06ef4152d59b97da1186cdf288588d20d69a689fe75b2140130fa5d8326d07ad"
+desktop_file_sum="1df6a4d5796e0435fe95b6e169c686e180d440b883d9076125a48a94f8119489"
+
+prepare() {
+  wget -qO "pycharm.desktop" "https://raw.githubusercontent.com/pacstall/pacstall-programs/master/packages/${name}/PyCharm.desktop"
+  if [[ "$(sha256sum pycharm.desktop | cut -d' ' -f 1)" != "${desktop_file_sum}" ]]; then
+    fancy_message error "Checksum for PyCharm.desktop failed"
+    return 1
+  fi
+}
+
+build() {
+  true
+}
+
+install() {
+  sudo mkdir -p "${STOWDIR}/${name}/usr/share/licenses/pycharm"
+  sudo mkdir -p "${STOWDIR}/${name}/usr/share/pycharm"
+  sudo mkdir -p "${STOWDIR}/${name}/usr/share/pixmaps"
+  sudo cp -dr --no-preserve='ownership' bin jbr lib plugins "${STOWDIR}/${name}/usr/share/pycharm/"
```

Comment:
To make Pycharm launch-able from terminal, make a symbolic link from `${STOWDIR}/${name}/usr/share/pycharm/bin/pycharm.sh` to `${STOWDIR}/${name}/usr/bin/pycharm` And also, this would then mean the `Exec` can be just `pycharm`.